specification/cognitiveservices/OpenAI.Inference/tspconfig.yaml

```diff
@@ -21,9 +21,9 @@ options:
     enable-sync-stack: true
     generate-tests: false
   "@azure-tools/typespec-ts":
-    emitter-output-dir: "{project-root}/generated"
     generateMetadata: false
     generateTest: false
     isModularLibrary: true
+    relative-generated-folder: "sources/generated"
```

Comment:
Is this key only introduced for JS emitter? Can JS emitter leverage `package-dir`? This is the latest schema we're aligned and is rolling out.
https://github.com/Azure/azure-rest-api-specs/pull/23866

Reply:
Yes, it is being added in https://github.com/Azure/azure-sdk-for-js/pull/26138.

Reply:
I added the comment in the above PR. //CC: @weshaggard

Reply:
Yes lets try to avoid this if possible. I'd prefer we simply append "sources/generated" to the package-dir path we are already computing instead of having folks need this property in their tspconfig file.